utils/compare.sh

```diff
@@ -7,7 +7,7 @@ install_touchstone() {
   touchstone_tmp=$(mktemp -d)
   python3 -m venv ${touchstone_tmp}
   source ${touchstone_tmp}/bin/activate
-  pip3 install -qq git+https://github.com/cloud-bulldozer/benchmark-comparison.git
+  pip3 install -qq git+https://github.com/mohit-sheth/benchmark-comparison.git
```

Comment:
Should we get any changes that are on your fork into the main repo of benchmark-comparison, so we don't need this line change and keep up to date with other changes that get into the main repo?

Reply:
https://github.com/cloud-bulldozer/benchmark-comparison/pull/62 was merged
changing it back to upstream master